Source/Shared/BPConfiguration.m

```diff
@@ -49,7 +49,7 @@
         "On which device to run the app."},
     {'c', "config", BP_MASTER | BP_SLAVE, NO, NO, required_argument, NULL, BP_VALUE, "configFile",
         "Read options from the specified configuration file instead of the command line"},
-    {'t', "test-bundle-path", BP_SLAVE, YES, NO, required_argument, NULL, BP_VALUE | BP_PATH, "testBundlePath",
+    {'t', "test-bundle-path", BP_MASTER | BP_SLAVE, YES, NO, required_argument, NULL, BP_VALUE | BP_PATH, "testBundlePath",
```

Comment:
@oliverhu I fixed this change.  I also had to fix a unit test to match the req arg changing from YES to NO.  Let me know if that's incorrect.